airflow/ui/src/pages/Dag/Details.tsx

```diff
@@ -0,0 +1,138 @@
+                    <ClipboardIconButton />
+                  </ClipboardRoot>
+                </HStack>
+              </Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Description</Table.Cell>
+              <Table.Cell>{dag.description}</Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Timezone</Table.Cell>
+              <Table.Cell>{dag.timezone}</Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>File Location</Table.Cell>
+              <Table.Cell>
+                <Code>{dag.fileloc}</Code>
+              </Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Last Parsed</Table.Cell>
+              <Table.Cell>
+                <Time datetime={dag.last_parsed} />
+              </Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Latest Dag Version</Table.Cell>
+              <Table.Cell>
+                <DagVersionDetails dagVersion={dag.latest_dag_version} />
+              </Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Start Date</Table.Cell>
+              <Table.Cell>
+                <Time datetime={dag.start_date} />
+              </Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>End Date</Table.Cell>
+              <Table.Cell>
+                <Time datetime={dag.end_date} />
+              </Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Last Expired</Table.Cell>
+              <Table.Cell>
+                <Time datetime={dag.last_expired} />
+              </Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Concurrency</Table.Cell>
+              <Table.Cell>{dag.concurrency}</Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Has Task Concurrency Limits</Table.Cell>
+              <Table.Cell>{dag.has_task_concurrency_limits}</Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Dag Run Timeout</Table.Cell>
+              <Table.Cell>{dag.dag_run_timeout}</Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Max Active Runs</Table.Cell>
+              <Table.Cell>{dag.max_active_runs}</Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Max Active Tasks</Table.Cell>
+              <Table.Cell>{dag.max_active_tasks}</Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Max Consecutive Failed Dag Runs</Table.Cell>
+              <Table.Cell>{dag.max_consecutive_failed_dag_runs}</Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>Catchup</Table.Cell>
+              <Table.Cell>{dag.catchup}</Table.Cell>
```

Comment:
Boolean values seem to be not rendered. Similar case for `has_task_concurrency_limits` .

https://stackoverflow.com/questions/38337262/cannot-render-boolean-value-in-jsx

Reply:
Good catch! I fixed for catchup, that seems to be the only boolean value we're currently trying to render.

Reply:
has_task_concurrency_limits is also a Boolean in the component .